sgkit/io/vcf/vcf_reader.py

```diff
@@ -705,6 +716,15 @@ def vcf_to_zarr(
         field_defs=field_defs,
     )
 
+    # Issue a warning if max_alt_alleles caused data to be dropped
+    ds = load_dataset(output)
+    max_alt_alleles_seen = ds.attrs["max_alt_alleles_seen"]
+    if max_alt_alleles_seen > max_alt_alleles:
```

Comment:
I think it's worth noting in the docstring for `max_alt_alleles` that this is only applicable to the array of base changes defining the alleles, and not any of the actual call data.

Reply:
Good point - I've updated the doc.